spec/controllers/json_collection_pagination_spec.rb

```diff
@@ -119,7 +122,7 @@ class FakeController < ApplicationController
     it 'allows pagination to be disabled' do
       data = object.send(:paginated_json_collection, collection, path_helper, 'id', 'desc', 0, '∞', false, {})
       expect(data[:json].items.count).to eq 15
-      expect(data[:meta][:per_page]).to eq '∞'
+      expect(data[:meta][:per_page]).to eq false
```

Comment:
I think Dispatcher and some other clients are expecting the response of  '∞'. Would it be possible to keep that unchanged?